airflow/providers/trino/hooks/trino.py

```diff
@@ -138,9 +138,17 @@ def get_pandas_df(self, hql, parameters=None, **kwargs):
             df = pandas.DataFrame(**kwargs)
         return df
 
-    def run(self, hql, autocommit: bool = False, parameters: Optional[dict] = None, handler=None) -> None:
+    def run(
+        self,
+        hql: str,
```

Comment:
We really should get rid of the hql across the whole class
Trino expecting sql. https://github.com/apache/airflow/issues/21597